compact/tracker.xml

```diff
@@ -71,13 +71,13 @@
     <constant name="TrackerEndcapNDisk1_zmin"        value="TrackerEndcapPDisk1_zmin" />
     <constant name="TrackerEndcapNDisk1_rmin"        value="TrackerEndcapPDisk1_rmin" />
     <constant name="TrackerEndcapNDisk1_rmax"        value="TrackerEndcapPDisk1_rmax" />
-    <constant name="TrackerEndcapNDisk2_zmin"        value="TrackerEndcapPDisk2_zmin" />
+    <constant name="TrackerEndcapNDisk2_zmin"        value="65*cm" />
     <constant name="TrackerEndcapNDisk2_rmin"        value="TrackerEndcapPDisk2_rmin" />
     <constant name="TrackerEndcapNDisk2_rmax"        value="TrackerEndcapPDisk2_rmax" />
-    <constant name="TrackerEndcapNDisk3_zmin"        value="TrackerEndcapPDisk3_zmin" />
+    <constant name="TrackerEndcapNDisk3_zmin"        value="90*cm" />
     <constant name="TrackerEndcapNDisk3_rmin"        value="TrackerEndcapPDisk3_rmin" />
     <constant name="TrackerEndcapNDisk3_rmax"        value="TrackerEndcapPDisk3_rmax" />
```

Comment:
Will the rmin (and rmax) stay the same even if you change the zmin?

Reply:
For now I don't have beampipe radii at those new location so I kept the previous number. So those three disks at electron direction may have a few milimeter larger than necessary opening around the beampipe but should not break anything.

Reply:
I updated rmin to reflect the new z positions.